app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java

```diff
@@ -54,16 +55,24 @@
     private String searchText = "";
 
     private SearchView searchView;
-
+    private TextView noTracks;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noTracks' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406594561/issues/source?bid=3233850&fileBranchId=3233896#l58)